src/components/settings-menu/UserDevSection.js

```diff
@@ -60,6 +61,7 @@ const UserDevSection = props => {
         </Column>
       </ListItem>
       {testnetsEnabled && <NetworkSection {...props} />}
+      <ListItem label="💥 Clear local storage" onPress={AsyncStorage.clear} />
```

Comment:
wait hold up...
```suggestion
      <ListItem label="💥 Clear local storage" onPress={AsyncStorage.clear()} />
```

Reply:
no

Reply:
is this not the correct way to call this function? https://react-native-async-storage.github.io/async-storage/docs/api/#clear

Reply:
or is `onPress={AsyncStorage.clear}` the same as?
```suggestion
      <ListItem label="💥 Clear local storage" onPress={() => AsyncStorage.clear()} />
```

Reply:
https://stackoverflow.com/questions/40275154/react-native-listitem-onpress-why-is-this-function-executing

Reply:
yes the first suggestion was wrong. haven't seen this syntax without using an arrow function before. TIL